### ISSUE TITLE
Better handling of multiple expiry dates of the same product

### ISSUE THREAD

**Issue by bpcurse** [lentschi/expiry_sync#14]
If you have e.g. 5 items of product x and 3 of them expire in 10 days and 2 of them expire in 16 days, it should somehow be possible to set different best before dates, without creating separate product copies.

**Comment by sap7772000zilla** [comment]
Hello.
Somehow...how? 
To me it makes perfect and practical sense to have 2 entries in this case.
You could store them separately or optically by drawing a sign on the label with a visible marker so that the picture will tell them apart.
Thank you.

**Comment by bpcurse** [comment]
@sap7772000zilla Yes, you're right. Sorry for being unprecise. Here's a more detailed proposal:

- In the edit article view add a button ("+") e.g. to the right of the expiry date which creates a new batch with independent count and expiry to the same product.
EDIT: The button could also be located as "create new batch" at the bottom alike "take another picture" or "copy article".
- Show this second batch by repeating the expiry and count rows below the first set.
- Add a "delete batch" button ("-") on the right side to the subsequent batches and maybe mark the following batches as batch 2, etc.
- In product overview show the sum of the quantities of all batches per article and the soonest expiry date OR show the individual counts and expiry dates.
- Alter behaviour of the add product button ("+") in the overview if a single product was selected by creating a new batch to this product and offer direct editing of quantity and expiry.
- Alter database structure accordingly, if necessary.

**Comment by sap7772000zilla** [comment]
Hi.
Let me be very clear here, I not a programmer so my opinion won't count for future development.
That being said I sorry to say but I still don't see significant value for this feature. 
I probably manage ~100 items and 2/3 of them are cans. Out of them I have 2 items with several pieces and different expiry dates meaning I have now 4 separate entries - completely insignificant compared with the whole IMO.
Also, let's accept that this app is meant for individuals and not stores / warehouses where, I agree, this feature might be better used.
Just out of curiosity, how many items do you manage with this app?
